tests/unit/processor/generic_adder/test_generic_adder_rule.py

```diff
@@ -1,8 +1,10 @@
 # pylint: disable=missing-docstring
+from typing import Hashable
 import pytest
 
 from logprep.processor.generic_adder.rule import GenericAdderRule
 from logprep.filter.lucene_filter import LuceneFilter
+from logprep.processor.base.exceptions import InvalidRuleDefinitionError
```

Comment:
Remove unused imports please.
```suggestion
import pytest

from logprep.processor.generic_adder.rule import GenericAdderRule
```

Reply:
done